bin/seed-salesforce

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+'use strict';
+require('dotenv').config();
+
+const moment = require('moment');
+const times = require('lodash/times');
+
+const mockAwardsForAll = require('../controllers/apply/under10k/mocks');
+const formBuilder = require('../controllers/apply/under10k/form');
+const salesforceService = require('../controllers/apply/form-router/lib/salesforce');
+
+const submitMockApplication = async (applicationId) => {
+
+    const dummyData = mockAwardsForAll.mockResponse({
+        projectLocation: 'bournemouth'
```

Comment:
The Covid-19 GMS sandbox is configured to only process applications in this location, FYI!